docs: update fusionreport score calculation documentation

KuthuruTejaswini opened the pull request:
Resolves #555

Updates documentation to reflect new fusionreport score calculation:
- Added detailed explanation of new scoring system
- Documented 80/20 weight distribution between tool detection and database hits
- Added formula for score calculation

## PR checklist

- [x] This comment contains a description of changes (with reason)
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If necessary, also make a PR on the [nf-core/rnafusion branch on the nf-core/test-datasets repo](https://github.com/nf-core/test-datasets/pull/new/nf-core/rnafusion)
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Make sure your code lints (`nf-core lint .`).
- [x] Documentation in `docs` is updated
- [ ] `CHANGELOG.md` is updated
- [ ] `README.md` is updated

**Learn more about contributing:** [CONTRIBUTING.md](https://github.com/nf-core/rnafusion/tree/master/.github/CONTRIBUTING.md)

alanmmobbs93 committed:
update test for skipping fastp

alanmmobbs93 committed:
update changelog

atrigila committed:
fix: change hardcoded container for seqera container, mv conda to environment.yml

atrigila committed:
docs: update changelog

atrigila committed:
fix: specify version 1.33 in bioconda and seqera container

alanmmobbs93 committed:
add new test

alanmmobbs93 committed:
Merge pull request #572 from alanmmobbs93/trim_workflow_test

Add nf-test to TRIM_WORKFLOW

nschcolnicov committed:
Merge pull request #550 from nf-core/general_fixes

Updated tests and fusion containers and conda versions, general confi…

atrigila committed:
refactor: emit a directory and add meta

atrigila committed:
test: add fusionreport nf-test

alanmmobbs93 committed:
update module

alanmmobbs93 committed:
update subworkflow

apeltzer committed:
Merge pull request #574 from alanmmobbs93/update_arriba-arriba_module

Update arriba_arriba module

rannick committed:
update fusioncatcher container

rannick committed:
Merge branch 'refactor-references' of https://github.com/nf-core/rnafusion into refactor-references

atrigila committed:
test: add test to `FUSIONREPORT_DETECT`

atrigila committed:
docs: update changelog

alanmmobbs93 committed:
change module structure and first test attempt

alanmmobbs93 committed:
update test and generate first snap

rannick committed:
fix merge conflicts

rannick committed:
fix linting issues

rannick committed:
prettier

alanmmobbs93 committed:
update tests after fixing fasta in github

alanmmobbs93 committed:
update changelog

atrigila committed:
Merge pull request #573 from atrigila/fusioncatcher_download

Change `FUSIONCATCHER_DOWNLOAD` docker image to Seqera container

atrigila committed:
docs: udpdate changelog

atrigila committed:
Merge branch 'dev' into fusionreport_detect

atrigila committed:
test: update nf-test

alanmmobbs93 committed:
update test excluding samtools

alanmmobbs93 committed:
Merge branch 'dev' into add_test_arriba_workflow
merge dev branch

atrigila committed:
Merge pull request #577 from atrigila/fusionreport_detect

Add nf-test to `FUSIONREPORT_DETECT`

rannick committed:
update comments

rannick committed:
Merge branch 'dev' into refactor-references

alanmmobbs93 committed:
update test to include cram files

alanmmobbs93 committed:
Merge branch 'dev' into add_test_arriba_workflow

alanmmobbs93 committed:
remove original script

alanmmobbs93 committed:
fix changelog

atrigila committed:
fix: fix container versions

atrigila committed:
test: add nf-test initial draft

rannick committed:
fix i/o in channels

rannick committed:
Merge branch 'refactor-references' of https://github.com/nf-core/rnafusion into refactor-references

rannick committed:
test build removed from ci as build_references happens before the run

rannick committed:
remove double nested folder

rannick committed:
update changelog

rannick committed:
update changelog

rannick committed:
update snapshots, add meta

rannick committed:
remove trash files

rannick committed:
update test

rannick committed:
update download snapshot

rannick committed:
update detect snapshot

alanmmobbs93 committed:
restructure module

rannick committed:
use stubs

rannick committed:
update changelog, first step

rannick committed:
update GTF_TO_REFFLAT options, update snapshots

rannick committed:
update star in snapshot

rannick committed:
Merge pull request #581 from nf-core/fix_fusionreport

Fix FUSIONREPORT_DOWNLOAD: remove double nested folder

atrigila committed:
fix: simplify starfusion module

atrigila committed:
tests: update nf-test starfusion

atrigila committed:
refactor: add stubs and ext.args

atrigila committed:
docs: update meta.yml

atrigila committed:
test: update snapshots

atrigila committed:
refactor: revert add nf-test initial draft

rannick committed:
add human_gencode_filter to starfusion build

rannick committed:
Merge branch 'dev' into refactor-references

rannick committed:
remove test.xml

rannick committed:
Merge branch 'refactor-references' of https://github.com/nf-core/rnafusion into refactor-references

rannick committed:
update snap

rannick committed:
try updating snapshot again

atrigila committed:
fix: add versions to dependencies

atrigila committed:
feat: add fusion annot lib as an external param

atrigila committed:
docs: add param type and where to get it

atrigila committed:
test: run starfusion in test_build

atrigila committed:
refactor: add metamap to output

atrigila committed:
test: update tests

atrigila committed:
test: add fusion_annot_lib so that stub does not fail

alanmmobbs93 committed:
init test with local reference

alanmmobbs93 committed:
PR changes

alanmmobbs93 committed:
Merge branch 'dev' into add_test_arriba_workflow

alanmmobbs93 committed:
create test with full reference locally

alanmmobbs93 committed:
Merge pull request #578 from alanmmobbs93/add_test_arriba_workflow

Add nf-test for local subworkflow `ARRIBA_WORKFLOW`

nf-core-bot committed:
Template update for nf-core/tools version 3.1.0

atrigila committed:
fix: add dependencies versions and update container

atrigila committed:
refactor: add readlength as ext.args

atrigila committed:
test: update stub structure

atrigila committed:
test: add nf-test to `starfusion`

atrigila committed:
docs: update changelog

atrigila committed:
Merge branch 'dev' into starfusion_detect

nvnieuwk committed:
fix test config resources

nvnieuwk committed:
add ctatsplicing module

rannick committed:
avoid using meta for remapping

nvnieuwk committed:
implement ctatsplicing

rannick committed:
fix merge conflicts

nvnieuwk committed:
fix pre-commit

nvnieuwk committed:
Update nextflow.config

Co-authored-by: Annick Renevey <47788523+rannick@users.noreply.github.com>

nvnieuwk committed:
update changelog

nvnieuwk committed:
Merge branch 'new-module-ctat-splicing' of github.com:nf-core/rnafusion into new-module-ctat-splicing

rannick committed:
use docker containers for fusioninspector, fusioncatcher and starfusion

rannick committed:
update changelog

rannick committed:
revert change

rannick committed:
revert change

nvnieuwk committed:
make ctatsplicing possible with arriba input

rannick committed:
test to see if dependencies are working with starfusion1.14 onluy

rannick committed:
use custom container for fusioncatcher, fix typo

rannick committed:
add dependencies explicitly

rannick committed:
update bin path

rannick committed:
Merge branch 'dev' into fusioninspector

rannick committed:
bin path to prep_genome_lib not working with singularity wave container, reverting to galaxy and specifying path

rannick committed:
propagate container of starfusion to fusioninspector

rannick committed:
update star

rannick committed:
update changelog

rannick committed:
update changelog

atrigila committed:
fix: update starfusion container declaration to latest nf-core standards

alanmmobbs93 committed:
updated test

alanmmobbs93 committed:
Merge branch 'dev' into add_test_fusioncatcher

nvnieuwk committed:
add some views to debug the full run

rannick committed:
update dependencies

rannick committed:
update path again

rannick committed:
add minimap

rannick committed:
propagate container/env change

rannick committed:
Merge pull request #588 from nf-core/add_singularity_containers

Add singularity containers

rannick committed:
Merge branch 'dev' into fusioninspector

nvnieuwk committed:
try to fix issue with arriba output

atrigila committed:
Update conf/modules.config

Co-authored-by: Annick Renevey <47788523+rannick@users.noreply.github.com>

nvnieuwk committed:
remove views

rannick committed:
fix merge conflicts

nvnieuwk committed:
remove breaking ifEmpties

rannick committed:
update changelog

nvnieuwk committed:
remove def

rannick committed:
update changelog, retrofit trim_workflowo

nvnieuwk committed:
redo star changes

nvnieuwk committed:
use bam from starfusion

rannick committed:
use placeholder and add TODO for update

nvnieuwk committed:
create junctions with arriba for ctatsplicing

rannick committed:
Merge branch 'dev' into starfusion_detect

nvnieuwk committed:
do not create an index in ctat splicing

atrigila committed:
fix: add full path to module to avoid changing nf-tests

atrigila committed:
feat: use previous `.dat.gz` file as default

atrigila committed:
Merge branch 'fusioninspector' of https://github.com/atrigila/rnafusion into fusioninspector

atrigila committed:
test: update snapshot

alanmmobbs93 committed:
update snaps

atrigila committed:
Merge pull request #583 from atrigila/fusioninspector

Add nf-test to `STARFUSION_BUILD` and refactor module

atrigila committed:
Merge branch 'dev' into starfusion_detect

atrigila committed:
feat: allow to use either reads or junction files

atrigila committed:
test: update snaps

atrigila committed:
docs: update changelog

atrigila committed:
fix: add junction if provided

nvnieuwk committed:
readd indexing to ctat & sort arriba bam

nvnieuwk committed:
Merge branch 'dev' into new-module-ctat-splicing

rannick committed:
fix merge conflicts

rannick committed:
add species as parameter

nvnieuwk committed:
remove ctatsplicing check from starfusion

nvnieuwk committed:
clarify changelog

nvnieuwk committed:
update docs

nvnieuwk committed:
prettyyy

rannick committed:
fix some issues with channels

rannick committed:
add species to schema

rannick committed:
fix erroneous addition of meta

nvnieuwk committed:
Merge pull request #587 from nf-core/new-module-ctat-splicing

New module ctat splicing

rannick committed:
Merge branch 'dev' into refactor-references

rannick committed:
fixes

nf-core-bot committed:
Template update for nf-core/tools version 3.1.1

rannick committed:
Merge branch 'dev' into nf-core-template-merge-3.1.1

atrigila committed:
Update modules/local/starfusion/detect/main.nf

Co-authored-by: Annick Renevey <47788523+rannick@users.noreply.github.com>

atrigila committed:
Update modules/local/starfusion/detect/main.nf

Co-authored-by: Annick Renevey <47788523+rannick@users.noreply.github.com>

atrigila committed:
Merge branch 'dev' into starfusion_detect

atrigila committed:
Merge pull request #586 from atrigila/starfusion_detect

Add nf-test to local module `STARFUSION_DETECT`

rannick committed:
Merge branch 'dev' into refactor-references

rannick committed:
Merge pull request #505 from nf-core/refactor-references

Refactor references

atrigila committed:
test: add nf-test for starfusion subworkflow

atrigila committed:
chore: add todo message

atrigila committed:
fix: remove commited debug view

atrigila committed:
docs: update changelog

atrigila committed:
chore: remove unused module `starfusion_downloadgenome`

atrigila committed:
docs: update changelog

atrigila committed:
Merge pull request #597 from atrigila/sbwf_starfusion

Add nf-test for local subworkflow `STARFUSION_WORKFLOW`

atrigila committed:
Merge branch 'dev' into remove_starfusion_download

atrigila committed:
feat: update fusioinspector to 2.10.0

atrigila committed:
Merge pull request #599 from atrigila/remove_starfusion_download

Remove local module `STARFUSION_DOWNLOAD`

atrigila committed:
fix: update container with missing dependencies

atrigila committed:
test: add full path to avoid impacting nf-test

atrigila committed:
test: add nf-test

atrigila committed:
fix: emit log only and not all outputs

atrigila committed:
fix: update dependencies in container

atrigila committed:
test: add config and update test

atrigila committed:
docs: update changelog

atrigila committed:
Merge branch 'dev' into fusioninspector_nftest

atrigila committed:
fix: add missing script

atrigila committed:
docs: update changelog

atrigila committed:
chore: remove unused module `bedops/convert2bed`

atrigila committed:
fix: remove hardcoded script

atrigila committed:
Merge branch 'fusioninspector_nftest' of https://github.com/atrigila/rnafusion into fusioninspector_nftest

atrigila committed:
fix: python script does not take flags

atrigila committed:
chore: add fixed version of dependencies

atrigila committed:
test: update tests

atrigila committed:
Merge pull request #603 from atrigila/issue_602

Add missing script in local module `GET_RRNA_TRANSCRIPTS`

atrigila committed:
test: add build test

atrigila committed:
chore: allow to run build_references only

atrigila committed:
tests: update md5 ignore list

atrigila committed:
test: update nf-test

atrigila committed:
feat: add all outputs

atrigila committed:
test: test all outputs

atrigila committed:
Merge branch 'fusioninspector_nftest' of https://github.com/atrigila/rnafusion into fusioninspector_nftest

atrigila committed:
test: add test_build to ci

atrigila committed:
test: update nf-test to capture stable and unstable files

atrigila committed:
test: update test and snapshot

atrigila committed:
chore: add indentation

atrigila committed:
test: add nf-test

alanmmobbs93 committed:
Merge branch 'dev' into add_test_fusioncatcher

alanmmobbs93 committed:
update snaps after merging

alanmmobbs93 committed:
remove bakcup file and update changelog

alanmmobbs93 committed:
fix bug with fusioncatcher ref

alanmmobbs93 committed:
pre-commit changes

alanmmobbs93 committed:
prettier changes

rannick committed:
Merge branch 'dev' into nf-core-template-merge-3.1.1

rannick committed:
fix some merge conflicts

rannick committed:
fix merge conflict

rannick committed:
update softwareversion file

rannick committed:
update version

alanmmobbs93 committed:
Merge pull request #591 from alanmmobbs93/add_test_fusioncatcher

Add nf-test for `FUSIONCATCHER` subworkflow

rannick committed:
Merge branch 'dev' into nf-core-template-merge-3.1.1

rannick committed:
Merge pull request #596 from nf-core/nf-core-template-merge-3.1.1

Important! Template update for nf-core/tools v3.1.1

rannick committed:
Merge branch 'dev' into fusioninspector_nftest

atrigila committed:
feat: update `fusion-report` to 3.1.1

atrigila committed:
test: add required params

atrigila committed:
Merge branch 'dev' into create_nftest_build_references

atrigila committed:
test: adapt the test to new template

atrigila committed:
test: try fix on template merge issue

atrigila committed:
Merge pull request #605 from atrigila/create_nftest_build_references

Add pipeline-level nf-test (`BUILD_REFERENCES`)

atrigila committed:
Merge pull request #601 from atrigila/fusioninspector_nftest

Add nf-test for local module `FUSIONINSPECTOR`

atrigila committed:
Merge branch 'dev' into fusionreport_workflow_nftest

atrigila committed:
test: update tests and snapshot

atrigila committed:
docs: update changelog

nf-core-bot committed:
Template update for nf-core/tools version 3.1.2

rannick committed:
Merge branch 'dev' into nf-core-template-merge-3.1.2

rannick committed:
remove md5sums for databases

rannick committed:
move outdir up

rannick committed:
update changelog

rannick committed:
excude databases subject to quick changes

rannick committed:
Merge pull request #609 from nf-core/nf-core-template-merge-3.1.2

Important! Template update for nf-core/tools v3.1.2

rannick committed:
Merge branch 'dev' into error_output_missing

rannick committed:
Merge branch 'dev' into fusionreport_workflow_nftest

rannick committed:
Merge pull request #611 from nf-core/error_output_missing

Fix error when output is missing

rannick committed:
Merge branch 'dev' into fusionreport_workflow_nftest

atrigila committed:
fix: add unstable files to nftignore

atrigila committed:
Merge pull request #607 from atrigila/fusionreport_workflow_nftest

Add nf-test for `fusionreport_workflow` subworkflow

nf-core-bot committed:
Template update for nf-core/tools version 3.2.0

rannick committed:
Merge branch 'dev' into nf-core-template-merge-3.2.0

rannick committed:
Merge pull request #615 from nf-core/nf-core-template-merge-3.2.0

Important! Template update for nf-core/tools v3.2.0

KuthuruTejaswini commented:
This PR addresses the documentation updates requested in that issue #555

atrigila commented:
Hi @KuthuruTejaswini, could you make this PR against the `dev` branch of https://github.com/nf-core/rnafusion? Currently it is against rannick's fork. Thank you.

dacotah03 committed:
docs: update fusionreport score calculation documentation

- Add detailed explanation of new scoring system
- Document 80/20 weight distribution between tool detection and database hits
- Add formula for score calculation

dacotah03 committed:
Apply pre-commit fixes and update changelog
